test/integration/command/test_audit.py

```diff
@@ -105,11 +105,12 @@ def test_conan_audit_paths():
     assert "If you don't have a valid token, register at: https://audit.conan.io/register." in tc.out
 
 
+@pytest.mark.xfail(reason="This test is not working")
```

Comment:
@AbrilRBS I had to mark this as xfail, this was failing, at least locally.
And I was also concerned it was reaching out the real http api end point.

Reply:
cc @czoido who implemented this test (But the corruption check happens before the server is queried)

Reply:
Sorry for that, if the test had been working correctly, It would never have touched the endpoint. I'm proposing some changes to fix that in this PR, but I'll also make a PR to avoid that any test for this contacts with the endpoint.

Reply:
@memsharded I made these changes to avoid contacting the endpoint in tests in a similar way we do for conancenter, please check: https://github.com/conan-io/conan/pull/17968/commits/ccb753a6098fda09797dcb4cf06c8de520dde864

Reply:
Great!